Find a functionally equivalent code:android.graphics.Matrix.preRotate:COMMENT
Method Modifier: public      
Comment:/**
 * Preconcats the matrix with the specified rotation.
 * M' = M * R(degrees)
 */

@@ -1,3 +1,4 @@
 {
-    return native_preRotate(native_instance, degrees);
+    native_preRotate(native_instance, degrees);
+    return true;
 }

